examples/upgradeable/v1/src/contract.rs

```diff
@@ -0,0 +1,35 @@
+use soroban_sdk::{
```

Comment:
since we have many examples for the upgradeable, a comment on top of this file, explaining its purpose and the diff between `v2` (or how they work together) would be good

Reply:
the same goes for `v2` file in examples